fbw-common/src/wasm/systems/systems/src/communications/amu.rs

```diff
@@ -0,0 +1,840 @@
+    Word15,
+    Word16,
+}
+
+#[repr(u32)]
+#[derive(FromPrimitive, Eq, PartialOrd, PartialEq, Copy, Clone)]
+pub enum LabelWordAMUACP {
+    Label300Request = 3,
+    Label301AMU = 0x83,
+    Label210VolumeControlVHF = 0x11,
+    Label211VolumeControlHF = 0x91,
+    Label212VolumeControlADFPA = 0x51,
+    Label213VolumeControlVORMKR = 0xD1,
+    Label215VolumeControlINTCAB = 0xB1,
+    Label217VolumeControlILS = 0xF1,
+}
+#[derive(Copy, Clone)]
+pub struct MixedAudio {
+    receive_com1: bool,
+    receive_com2: bool,
+    receive_adf1: bool,
+    receive_adf2: bool,
+    receive_vor1: bool,
+    receive_vor2: bool,
+    receive_ils: bool,
+    enable_beep: bool,
+    receive_markers: bool,
+    sound_markers: bool,
+
+    // FOR FUTURE USE: Not needed for the time being as there's no K event for all this
+    // receive_gls: bool,
+    // receive_hf1: bool,
+    // receive_hf2: bool,
+    // receive_mech: bool,
+    // receive_att: bool,
+    // receive_pa: bool,
+    //
+    volume_com1: u32,
+    volume_com2: u32,
+    volume_com3: u32,
+    volume_adf1: u32,
+    volume_adf2: u32,
+    volume_vor1: u32,
+    volume_vor2: u32,
+    volume_ils: u32,
+    // FOR FUTURE USE: Not needed for the time being as there's no K event for all this
+    // volume_markers: u32,
+    // volume_hf1: u32,
+    // volume_hf2: u32,
+    // volume_att: u32,
+    // volume_mech: u32,
+    // volume_pa: u32,
+    //
+}
+impl MixedAudio {}
+impl Default for MixedAudio {
+    fn default() -> MixedAudio {
+        MixedAudio {
+            receive_com1: false,
+            receive_com2: false,
+            receive_adf1: false,
+            receive_adf2: false,
+            receive_vor1: false,
+            receive_vor2: false,
+            receive_ils: false,
+            enable_beep: false,
+            receive_markers: true,
+            sound_markers: false,
+            // FOR FUTURE USE: Not needed for the time being as there's no K event for all this
+            // receive_gls: false,
+            // receive_hf1: false,
+            // receive_hf2: false,
+            // receive_mech: false,
+            // receive_att: false,
+            // receive_pa: false,
+            //
+            volume_com1: 0,
+            volume_com2: 0,
+            volume_com3: 0,
+            volume_adf1: 0,
+            volume_adf2: 0,
+            volume_vor1: 0,
+            volume_vor2: 0,
+            volume_ils: 0,
+            // FOR FUTURE USE: Not needed for the time being as there's no K event for all this
+            // volume_markers: 0,
+            // volume_hf1: 0,
+            // volume_hf2: 0,
+            // volume_att: 0,
+            // volume_mech: 0,
+            // volume_pa: 0,
+            //
+        }
+    }
+}
+
+#[derive(Default, PartialEq, Eq)]
+pub enum AudioSwitchingKnobPosition {
+    Captain,
+    #[default]
+    Norm,
+    Fo,
+}
+
+pub struct Amu {
+    adaptation_board: AdaptationBoard,
+}
+impl Amu {
```

Comment:
For consistency can we rename this to `AudioManagementUnit` (or whatever this actually means?). If you give these really short names it's always a bit ambigious whether the type is `Amu`, oder `AMU`. Giving it a long name is easier to import, easier to search for and less ambigious with regards to some abbreviations that are used by different systems. Any finally, for consistency :-)

Reply:
Roger, you're right :)